src/test/kotlin/org/rust/ide/typing/paste/RsCopyPasteTestBase.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.typing.paste
+
+import com.intellij.codeInsight.CodeInsightSettings
+import org.rust.RsTestBase
+
+abstract class RsCopyPasteTestBase: RsTestBase() {
+    private var savedImportsOnPasteSetting: Int = 0
+
+    override fun setUp() {
+        super.setUp()
+        savedImportsOnPasteSetting = CodeInsightSettings.getInstance().ADD_IMPORTS_ON_PASTE
```

Comment:
Looks like it is java setting. Maybe add our checkbox to `Editor | General | Auto Import | Rust`